README.rst

```diff
@@ -13,14 +13,15 @@ It is a good way to just try out DeepSpeech, without much knowledge on how DeepS
 
 **Python:**
 
-* `Microphone VAD streaming  <mic_vad_streaming/README.rst>`_
-* `VAD transcriber  <vad_transcriber/README.rst>`_
+* `Microphone VAD streaming  <mic_vad_streaming/>`_
```

Comment:
Those changes do not look right ?

Reply:
On the master branch those links are all broken.  It looks like I checked out my branch off of master - should I have cut from the 0.6 branch instead?

Reply:
> On the master branch those links are all broken. It looks like I checked out my branch off of master - should I have cut from the 0.6 branch instead?

Yes, as I said in the comment below, but you might not yet have seen it since we replied at the same time :)